pkg/hubble/parser/threefour/parser.go

```diff
@@ -608,3 +620,50 @@ func getTCPFlags(tcp layers.TCP) string {
 
 	return strings.Join(info, comma)
 }
+
+func decodeDebugCapturePoint(dbg *monitor.DebugCapture) pb.DebugCapturePoint {
+	if dbg == nil {
+		return pb.DebugCapturePoint_DBG_CAPTURE_POINT_UNKNOWN
+	}
+	return pb.DebugCapturePoint(dbg.SubType)
+}
+
+func decodeNetworkInterface(tn *monitor.TraceNotify, dbg *monitor.DebugCapture) *pb.NetworkInterface {
+	ifIndex := 0
```

Comment:
nit: `ifIndex` could be `uint32` here, reducing a bit the casting dance following.